### DISCUSSION TITLE
Failing to pass binding from listing entity to popup modal (again!)

### DISCUSSION THREAD

**Discussion by danielrosehill** [Budibase/budibase#14040] [Help]
Any help appreciated!

Starting point - employees listing page, card block:

![image](https://github.com/Budibase/budibase/assets/40233203/55cd7965-3202-446f-a340-f04feb3ae492)

Created a button for each employee:

![image](https://github.com/Budibase/budibase/assets/40233203/ef8802ad-dd6c-4bdd-ab81-be42f54aa725)

Added a key `selected_employee` and added the first value to be setting that as a value and the second as opening a modal (tried the same thing with the side card popup and couldn't get it to work either):

![image](https://github.com/Budibase/budibase/assets/40233203/1a0d83d7-2f5e-4c48-a416-3c0e5d2ecb57)

Example string:

![image](https://github.com/Budibase/budibase/assets/40233203/d7bda7be-8491-4c71-8695-88661b8955a9)

As configured:

![image](https://github.com/Budibase/budibase/assets/40233203/488bb9e1-ec0a-49e4-ab46-7691ed1db870)

Filter value for modal = `{{ State.selected_employee}}`

![image](https://github.com/Budibase/budibase/assets/40233203/977fc199-bb62-4b32-9de2-3a62e9e4c33d)

Filter on that row in the popup:

![image](https://github.com/Budibase/budibase/assets/40233203/b506e57b-2650-4f50-a601-9b9495602030)

Popup either displays no rows or doesn't update as different buttons are clicked (ie, data is either not getting passed or not getting updated as expected)

**Comment by mikesealey** [comment]
Hey @danielrosehill 

When/where is your state set? I would try dropping in a "Show Notification" both before an after the "Update State" button. Typically, if an action is failing, it will stop working through the list of actions, so if only the notification before "Update State" displays, there may be an issue, and if neither show, it could be that the list of actions isn't being triggered when you expect.

Let me know how you get on, I'm sure it's something simple, but if it turns out to be a bug I'll dig a little deeper and maybe file a bug-report about it.
